mlflow/models/model.py

```diff
@@ -716,28 +715,34 @@ def load(cls, path) -> "Model":
             model2 = Model.load("s3://mybucket/path/to/my/model")
         """
         # Check if the path is a local directory and not remote
-        path_scheme = urlparse(str(path)).scheme
+        path = str(path)
+        path_scheme = urlparse(path).scheme
         if (not path_scheme or path_scheme == "file") and not os.path.exists(path):
             raise MlflowException(
                 f'Could not find an "{MLMODEL_FILE_NAME}" configuration file at "{path}"',
                 RESOURCE_DOES_NOT_EXIST,
             )
 
-        path = download_artifacts(artifact_uri=path)
-        if os.path.isdir(path):
-            path = os.path.join(path, MLMODEL_FILE_NAME)
-            env_var_path = os.path.join(path, ENV_VAR_FILE_NAME)
-        elif os.path.isfile(path):
-            env_var_path = os.path.join(os.path.dirname(path), ENV_VAR_FILE_NAME)
-        else:
+        is_model_dir = not path.endswith(MLMODEL_FILE_NAME)
```

Comment:
```suggestion
        is_model_dir = not path.endswith(f"/{MLMODEL_FILE_NAME}")
```

Reply:
Is it possible users just use "MLmodel"?